data-raw/model-parameters.R

```diff
@@ -272,10 +256,6 @@ params <- list(
   ..surv_juv_delta_contact_points = (.0358 * -0.189),
   ..surv_juv_delta_total_diverted = (.5 * -0.0021),
   ..surv_juv_outmigration_sj_int = -3.5,
-  ..surv_juv_outmigration_sac_int_one = 2.5,
```

Comment:
Removed these surv_juv_outmigration_sac params because we no longer use them after updating the Sac outmigration submodel based on Cyril new model.

Reply:
makes sense